test/cpp/naming/resolver_component_tests_runner_invoker.cc

```diff
@@ -55,6 +55,16 @@ ABSL_FLAG(
     "False indicates that this test is running under run_tests.py. "
     "Child process test binaries are located differently based on this flag. ");
 
+#ifdef GPR_WINDOWS
+ABSL_FLAG(bool, running_locally, false,
```

Comment:
More options here:
- You could potentially use the "GRPC_PORT_ISOLATED_RUNTIME=1" define, which is only set when running on RBE (normally it's used to determine the port picker behavior, but with an explanatory comment you could potentially use it for detecting whether running on RBE or no).
https://github.com/grpc/grpc/blob/c867c418c13652b8ece1986cff6717541f89f2c8/tools/remote_build/include/rbe_remote_execution.bazelrc#L31
- alternatively, you could set an --action_env or --test_env variable and add it to rbe_remote_execution.bazelrc (since detection whether running on RBE or not is a reasonable thing to detect)

Reply:
I have removed this flag since it's not needed anymore. We can just pass `--enable_runfiles=yes` to Bazel when run the test locally. See the below response.